source/guides/puppetlabs_package_repositories.markdown

```diff
@@ -18,33 +18,37 @@ Puppet Collections are numbered with integers, beginning with PC1. The higher th
 
 ### Yum-based Systems
 
-The [yum.puppetlabs.com](https://yum.puppetlabs.com) repository supports the following versions of RHEL and derivatives:
+The [yum.puppetlabs.com](https://yum.puppetlabs.com) repository supports the following versions of Red Hat Enterprise Linux and distributions based on it:
 
 {% include pup40_platforms_redhat_like.markdown %}
 
-{% include pup40_platforms_fedora.markdown %}
+{% include pup43_platforms_fedora.markdown %}
 
 To enable the repository, run the command below that corresponds to your OS version:
 
 #### Enterprise Linux 7
 
-    $ sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm
+    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm
 
 #### Enterprise Linux 6
 
-    $ sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-6.noarch.rpm
+    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-6.noarch.rpm
 
 #### Enterprise Linux 5
 
-    $ sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-5.noarch.rpm
+    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-5.noarch.rpm
 
-#### Fedora 20
+#### Fedora 22
 
-    $ sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-20.noarch.rpm
+    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-22.noarch.rpm
 
 #### Fedora 21
 
-    $ sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-21.noarch.rpm
+    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-21.noarch.rpm
+
+#### Fedora 20
+
+    sudo rpm -ivh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-20.noarch.rpm
```

Comment:
I think if it's EOL'd we should probably remove it... I would also think the package should be removed from yum.puppetlabs.com...